src/components/NotificationContext/NotificationContext.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { SnackbarSuccess } from '../Snackbars/SnackbarSuccess';
+import { SnackbarError } from '../Snackbars/SnackbarError';
+
+type NotificationContextType = {
+  notification: { type: string; message: string };
+  setNotification: React.Dispatch<React.SetStateAction<{ type: string; message: string }>> | undefined;
+};
+
+const NotificationContext = React.createContext<NotificationContextType>({
+  notification: {
+    type: '',
+    message: '',
+  },
+  setNotification: undefined,
+});
+
+export const useNotifications = (): NotificationContextType => {
+  const context = React.useContext(NotificationContext);
+
+  if (!context) {
+    throw new Error('useNotificationContext must be used within a NotificationProvider');
+  }
+
+  return context;
+};
+
+// ToDo type childrenn
+export default function NotificationContextComponent(props: { children: any }) {
```

Comment:
```
children: ReactNode
```

Reply:
The convention is to name it as `NotificationsProvider`

Reply:
done